### DISCUSSION TITLE
customizeable "target" at topics.html for using in <iframe> directives.

### DISCUSSION THREAD

**Discussion by paule32** [doxygen/doxygen#11199] [Ideas]
Hello,
I currently work on my own implementation of displaying the TOC (Table of Contents) with help of HTML < iframe directive, which can be used on HTML and CHM Project's.

Currently Doxygen provides/generate the following (Snippet) Code in the **topics.html**.
```
    <table class="directory">
        <tr id="row_0_"         class="even">
            <td class="entry">
                <span style="width:0px;display:inline-block;">&#160;</span>
                <span id="arr_0_" class="arrow" onclick="dynsection.toggleFolder('0_')">&#9660;</span>
                <a class="el" href="/miniFPC/group__fpc__compiler.html" target="_self">Der Free Pascal Compiler - FPC</a>
            </td><td class="desc"></td></tr>
```
Because the possibility, to use i-Frames, it is possible to structure the WebPage in different parts, that can act on it's own Pageinformation's in different Page targets.
I-Frames are the new < frameset way that is prefered, and suggested to use instead the old < frameset directives.

However, I decide to use i-Frame in inclusion with < div directive for two projects (HTML and CHM).
My Concerns are focused to the *target*.
Currently it is set to *_self*.
But I need a way (maybe on Doxygen Setting, which makes it possible to define targets when
**GENERATE_TREEVIEW** is set to YES.

**Comment by albert-github** [comment]
Did you try the doxygen command `\tableofcontents` for an inline TOC as also can be seen e.g. at https://www.doxygen.nl/manual/changelog.html on the right hand side of the page?
